custom/enikshay/const.py

```diff
@@ -169,3 +169,7 @@
 AGENCY_LOCATION_FIELDS = [
     ('private_sector_org_id', "Private Sector Org ID", []),
 ]
+
+DSTB_EPISODE_TYPE = "confirmed_tb"
+NEW_2B_APP_PERSON_CASE_VERSION = 20
```

Comment:
At some point this won't be `NEW` anymore. Might be easier to just call this `V20` or something.